test/etest/README.md

```diff
@@ -0,0 +1,256 @@
+
+# Introduction
+
+etest is an ERT extension to make it easier to test Emacs behaviour such as cursor and mark movement and inferior buffer output..
```

Comment:
British spelling & double period typo at the end. There's also britishism "initialisation" a few times later on

Reply:
Thanks but I think I prefer that etest uses the British/European spelling of English.

Reply:
Could you reconsider? The rest of Emacs uses American spelling for the sake of consistency.

Reply:
I think this is the sort of things that should be consistent within projects rather than across projects. But since you feel strongly about it, I have changed to American spelling.